largestPrimeFactor.js

```diff
@@ -1,6 +1,12 @@
-exports.largestPrimeFactor = function(n){
-  var primeNumber = 0;
+exports.largestPrimeFactor = function (n) {
+  let primeNumber = 2;
   // do your work here
-
+  while (primeNumber <= n) {
+    if (n % primeNumber == 0) {
```

Comment:
you probably want to wrap the if/else is another while loop that keeps pulling the same prime number out of n. So if n = 8, you keep pulling 2 out three times until n === 2.

Reply:
I think that that's how the code currently works. so if n === 8 then...
2 <= 8 {
  so 8 % 2 === 0 {
    8 /=2 --> evaluates to n=4 
(loop back to top and then do the same turning 4 into 2).

or am I misunderstanding?

Reply:
You're right. Approved!